### PR TITLE
Fixed typo "accross"

### PR DESCRIPTION
**Pull request by Bouke**

### CODE REVIEW

File: Sources/PackageLoading/PackageExtensions.swift [hunk 1]
```diff
@@ -102,7 +102,7 @@ extension Module.Error: FixableError {
         case .mixedSources(_):
             return "use only a single language within a module"
         case .duplicateModule(_):
-            return "modules should have a unique name, accross dependencies"
+            return "modules should have a unique name, across dependencies"
```
Comment (aciidb0mb3r) [inline Sources/PackageLoading/PackageExtensions.swift]: nit: is comma required here?

Comment (ddunbar) [inline Sources/PackageLoading/PackageExtensions.swift]: Nope

Comment (Bouke) [inline Sources/PackageLoading/PackageExtensions.swift]: Removed

**Comment by ddunbar** [comment]
Thanks!

**Pull request merged**